torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -90,11 +105,6 @@ def autotune_hints_to_configs(
     """
     xyz_options: tuple[tuple[int, Optional[int], Optional[int]], ...]
     configs: List[Config] = []
-    warp_size = device_props.warp_size
```

Comment:
This part got removed and not re-added. not sure if intentional

Reply:
was unused